### DISCUSSION TITLE
travellings 证书问题

### DISCUSSION THREAD

**Discussion by WindowsRegedit** [travellings-link/travellings#1328] [寻求帮助] [answered]
<img width="960" alt="image" src="https://user-images.githubusercontent.com/90195560/207198992-1d231018-749a-4a89-a056-ab8b014392f4.png">
如图，在今天访问https://status.travellings.link/status/index以及https://data.travellings.link/share/js3UvdC4/%E5%BC%80%E5%BE%80时，均出现cloudflare的“Invalid SSL Verification”问题.

**Comment by linlinzzo** [comment, marked as answer]
目前开往迁回国内，数据统计会定期在QQ群里公开，网站状态不再使用。
